tls/_common/_constructs.py

```diff
@@ -149,7 +149,7 @@ def EnumClass(type_field, type_enum):  # noqa
     return construct.SymmetricMapping(type_field, mapping)
 
 
-def EnumSwitch(type_field, type_enum, value_field, value_choices):  # noqa
+def EnumSwitch(type_field, type_enum, value_field, value_choices, default=construct.Switch.NoDefault):  # noqa
```

Comment:
Thanks a lot for the review! 

Yes re: the reason we have `# noqa` here.

1. Probably? I don't have any strong opinions about this, I might not have enough bandwidth to do a refactor/rename of these things _and_ work on the extensions at the moment, and if I had to pick one, I would pick the extensions work. But, feel free to file a ticket / send a PR, happy to review! :)

2. Yeah, makes sense. Updating.